cisagov/XFD
# Remove ability to add members to the members table in the Organization Details page

#  #

## 🗣 Description ##
In an individual organization's details, update the members table to remove the ability to add new members to it and modernize the table.



## 💭 Motivation and context ##

- Adding new members to organizations should not be done on this page.
- The members table should be updated to use the same table format as the rest of the tables.
- Additionally, added validation for when a member is to be removed from an organization so that you can't accidentally remove someone.

JIRA: CRASM-418





## 🧪 Testing ##

1. Log in as "Global Admin"
2. Go to "Manage Organizations"
3. Choose an organization and click on the "View/Edit" button
4. Click on "Members" Tab
5. See that "Add member" is removed from the members table and that the table is updated and functional





## 📷 Screenshots (if appropriate) ##
Original
![image](https://github.com/user-attachments/assets/01eda1aa-4df2-4bbf-a902-2c937c910038)

Updated
![image](https://github.com/user-attachments/assets/f8e499d5-e0da-4b89-9ba4-2a395a9d07cb)

Added member removal validation popup
![Are you sure you want to remove this user](https://github.com/user-attachments/assets/5e02b039-7ebd-4284-9e41-c4849b3b693b)
If removal fails
![Are you sure you want to remove this user](https://github.com/user-attachments/assets/2f6c8e6b-f520-4832-aa5c-d102aa6cf91f)
If removal is success
![MEMBERS](https://github.com/user-attachments/assets/c75a1378-65c9-4f16-929d-8850fe2ad350)


## ✅ Pre-approval checklist ##






      in code comments.
      to reflect the changes in this PR.

## ✅ Pre-merge checklist ##






## ✅ Post-merge checklist ##
